### ISSUE TITLE
Throws errors when binary file is in template folder

### ISSUE THREAD

**Issue by crutchcorn** [amwmedia/node-plop#70]
When a binary file such as a image file is located within the templates folder and `addMany` is used, it will throw a very cryptic error message as such: 
```
[FAILED] addMany Lexical error on line 1. Unrecognized text.
  ☺ ☻ ►►      h♦  &
^
```

A code example can be found here:
https://github.com/crutchcorn/generate-me/tree/14370feb32d05f913387929c7ac1359c52ed3b1a

By running `node index.js "Test" n y`, you can see the behavior

**Comment by amwmedia** [comment]
`addMany` is for adding files that are processed as `handlebars` templates. Maybe we could use something like [isBinaryFile](https://www.npmjs.com/package/isbinaryfile) to present a better error.

**Comment by amwmedia** [comment]
I think there's also room for a built-in `copy` action that would copy files without processing them as `templateFile`s.

**Comment by crutchcorn** [comment]
Certainly a better error should be implamented. However, I'm wondering if we should even ignore binary files in the built-in `copy` action. I'd argue that having binary files copied without errors is the most logical step given the rest of the libraries' behavior but you would have a much better understanding of the expected behavior as it's your project.

If you say one way or another, I'll start on a PR that will fix this issue

**Comment by amwmedia** [comment]
released with plop [v2.1.0](https://github.com/amwmedia/plop/releases/tag/v2.1.0)
